docs/basic-usage.rst

```diff
@@ -0,0 +1,82 @@
+Basic Usage
+###########
+
+Run Built Executable
+====================
+To get started, you must first run the built executable of CodeChain.
+
+In order to run CodeChain, run
+::
+
+    ./target/release/codechain
+
+Currently only operates in standalone mode, and you can create a block by sending a parcel through `JSON-RPC <https://github.com/CodeChain-io/codechain/wiki/JSON-RPC>`_ or `JavaScript SDK <https://api.codechain.io/>`_.
+
+Blockchain Configuration
+========================
+When configuring CodeChain's blockchain type, you can set it to either ``Solo``, ``Solo-Authority`` or ``Tendermint``. 

```

Comment:
Mention other consensus algorithm that will be supported soon

- HotStuff
- PoW
- PoA
-